rasa_addons/core/tracker_stores/botfront.py

```diff
@@ -249,8 +256,23 @@ def retrieve(self, sender_id):
 
         # the tracker exist localy an there is no new infos
         return self._convert_tracker(sender_id, current_tracker)
-
-    def sweep(self):
+    
+    def cleanup_test_trackers(self):
```

Comment:
this is basically a copy of that other method... just combine the two, keeping a different timeout constant for test trackers (self.test_tracker_persist_time) as needed